app/models/comment.rb

```diff
@@ -48,8 +48,8 @@ class Comment < ActiveRecord::Base
 
   # class methods .............................................................
   class << self
-    alias_method :by_user, :find_comments_by_user
-    alias_method :for_commentable, :find_comments_for_commentable
+    alias by_user find_comments_by_user
+    alias for_commentable find_comments_for_commentable
```

Comment:
Do these need to be aliases or could the original scopes be renamed?

Reply:
These scopes were brought in with the gem. I'd prefer not to change in case they have internal gem calls